deepfence_frontend/apps/dashboard/src/features/postures/pages/PostureCloudDetailModal.tsx

```diff
@@ -274,10 +294,77 @@ const DetailsComponent = ({
   );
 };
 
+const Content = () => {
+  const [isRemediationOpen, setIsRemediationOpen] = useState(false);
+
+  const {
+    data: { data: cloudPostures },
+  } = useGetComplianceDetails();
+  const data = cloudPostures.length ? cloudPostures[0] : undefined;
+
+  let prefix = '';
+  if (data?.cloud_provider?.toLowerCase().includes('aws')) {
```

Comment:
We may require such condition check in future, I think it would be better to have own utility. Because we had removed possible string and used enum.

Reply:
let's decide later if the code is repeated at many places, we will refactor it.